js/app.js

```diff
@@ -1,9 +1,117 @@
 // wait for DOM to load before running JS
 $(document).ready( function() {
 
-  // check to make sure JS is loaded
-  console.log('JS is loaded!');
+  // https://spotify-token-finder.herokuapp.com/
+  let query = $(".search-bar").val();
+  let type = $('.select-type').val();
 
-  // your code here
+  $(".submit").on("click", function(event) {
+    event.preventDefault();
+    query = $(".search-bar").val();
+    type = $(".select-type").val();
+    grabJson(query, type);
+  });
 
+  $(window).keydown(function(event) {
+      if (event.keyCode === 13) {
+          event.preventDefault();
+          query = $(".search-bar").val();
+          type = $(".select-type").val();
+          grabJson(query, type);
+      }
+  });
+
+  let grabJson = (query, type)=>
+    $.ajax({
+      type: "GET",
+      url: "https://api.spotify.com/v1/search",
+      data: {
+        q: query,
+
+          //check the api for the other types, currently, only album works
+        type: type,
+        // limit: 20
+      },
+      headers: {
+        Authorization: "Bearer BQBwkYoZwsRvGL2a47mVJP9E3LKjREs9OGAZoFjNQp6qoCbZK_USFvV_5u0Jgpbp_uSgRa4LmTIXl-EIhlmsBQ"
+      }
+    })
+    .then(function(response) {
```

Comment:
David,

Great job with this assignment. Your code is really good, I especially like that you broke out the code to render the html from the AJAX call and you thought about adding code if someone hit return instead of pressing the submit button on the form.

Keep up the good work!

Matt